docs/api/modules.md

```diff
@@ -253,91 +253,91 @@ ___
 
 ### IS\_ETH\_SIGNER
 
-• `Const` **IS\_ETH\_SIGNER**: ``"isEthSigner"``
+• **IS\_ETH\_SIGNER**: ``"isEthSigner"``
```

Comment:
Will these changes not be overwritten during build?

Reply:
@JGiter What do you mean?
I think you have a good question, though, this change was automatically added when I committed, but maybe it was a mistake.